specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devcenter.json

```diff
@@ -713,9 +713,12 @@
           {
             "name": "poolName",
             "in": "path",
-            "description": "Pool name",
+            "description": "The DevCenter Project upon which to execute operations.",
             "required": true,
-            "type": "string"
+            "type": "string",
```

Comment:
This appears in the new API version, and it's set via decorators. I don't know how to change it so the decorators only apply on the new API version.

Reply:
this is the correct for previous api as well, we just never added it. So you're actually making an improvement here